guides/hack/01-getting-started/02-tools.md

```diff
@@ -1,3 +1,47 @@
-**Under Construction**
+## Tools
 
-A placeholder for tools info.
+The core tools are:
+
+- `hh_client`: this is the command line interface for Hack's static analysis; it
+  is needed to verify that a project is valid Hack, and is used to find errors
+  in your programs
+- `hhvm`: this is used to execute your Hack code, and can either be used for
+  CLI (e.g. `hhvm foo.hack`) or as a server, and has
+  [extensive documentation](/hhvm/)
+
+### Editors and IDEs
+
+While you can edit Hack with any text editor, enhanced support is available
+for several.
```

Comment:
This sentence reads oddly to me. How about something like this?

> Whist you can edit Hack with any text editor, several editors are available with dedicated Hack functionality.

(Aside: Have we stopped mentioning Nuclide already?)

Reply:
Yep, Nuclide is dead as an open source project